deploy/lvm-operator.yaml

```diff
@@ -1182,6 +1182,7 @@ spec:
             - "--strict-topology"
             - "--leader-election"
             - "--extra-create-metadata=true"
+            - "--default-fstype=ext4"
```

Comment:
This file is autogenerated, you need to make the same change in deploy/yamls/zfs-driver.yaml

Reply:
done